### PR TITLE
create join

### PR DESCRIPTION
**Pull request by jinpark**

### CODE REVIEW

File: poker/views.py [hunk 1]
```diff
@@ -111,3 +116,18 @@ def user_action(request):
...
+@require_POST
+def join_game(request):
+    game_guid = request.POST.get("game_guid", None)
+    user_name = request.POST.get("name", None)
+    if not game_guid:
+        return _json_error_response("Add game guid")
+    if not user_guid:
+        return _json_error_response("Add name please")
+    game, created = Game.objects.get_or_create(guid=game_guid)
+    user = User(username=name, guid=uuid.uuid4())
```
Comment (eytoss) [inline poker/views.py]: no need to specify guid but it's ok.

Comment (eytoss) [inline poker/views.py]: name is undefined?

Comment (jinpark) [inline poker/views.py]: blah, yea i meant to put in `user_name`

File: poker/views.py [hunk 2]
```diff
@@ -111,3 +116,18 @@ def user_action(request):
...
     return _json_success_response("Action completed.")
+
+
+@require_POST
+def join_game(request):
+    game_guid = request.POST.get("game_guid", None)
+    user_name = request.POST.get("name", None)
+    if not game_guid:
+        return _json_error_response("Add game guid")
+    if not user_guid:
```
Comment (eytoss) [inline poker/views.py]: typo should be  if not user_name

**Pull request merged**